Library/Homebrew/utils/bottles.rb

```diff
@@ -11,7 +11,7 @@ module Bottles
     class << self
       # Gets the tag for the running OS.
       def tag(symbol = nil)
-        return Tag.from_symbol(symbol) if symbol.present?
+        return Tag.from_symbol(symbol.to_sym) if symbol.present?
```

Comment:
IMO this should be fixed a level higher.

Reply:
I thought that too (in part because I thought that might be able to fix it in one place instead of two), but I can't follow the call stack from the Sorbet error message. Happy to take a suggestion or replacement PR with a nicer fix.